### PR TITLE
test: add Linux package content verification

### PR DESCRIPTION
**Pull request by nr-ahemsath**
## Description

Adds Linux package content verification to the ArtifactBuilder.  A new dependency on 7Zip being installed in the environment the ArtifactBuilder is run in is created.  7Zip is installed on the Windows GHA runner the ArtifactBuilder runs on in CI.  Developers who need to use the ArtifactBuilder with the `LinuxPackages` argument will need to install 7Zip on their system.

# Author Checklist
- [ ] Unit tests, Integration tests, and Unbounded tests completed
- [ ] Performance testing completed with satisfactory results (if required)

# Reviewer Checklist
- [ ] Perform code review
- [ ] Pull request was adequately tested (new/existing tests, performance tests)

### CODE REVIEW

File: build/ArtifactBuilder/Artifacts/LinuxPackage.cs [hunk 1]
```diff
@@ -1,43 +1,187 @@
...
+            pInfo.RedirectStandardOutput = true;
+            pInfo.RedirectStandardError = true;
+            pInfo.UseShellExecute = false;
+
+            var process = Process.Start(pInfo);
+            process.WaitForExit(30000);
+            if (!process.HasExited)
+            {
+                process.Kill();
+                throw new Exception($"7z unpacakge failed to complete in a timely fashion.");
```
Comment (nrcventura) [inline build/ArtifactBuilder/Artifacts/LinuxPackage.cs:129]: `Unpackage` is misspelled here and in another location.

Comment (nr-ahemsath) [reply build/ArtifactBuilder/Artifacts/LinuxPackage.cs]: Good catch, thanks.

**Review by nrcventura — APPROVED** [review]

**Review by jaffinito — APPROVED** [review]

**Comment by chynesNR** [comment]
Are we sure we need to use 7zip? I though .NET added a zip decompression library around... Framework 4.5?

**Comment by chynesNR** [comment]
> Are we sure we need to use 7zip? I though .NET added a zip decompression library around... Framework 4.5?

My bad, this is the Linux PR.

**Review by chynesNR — APPROVED** [review]

**Pull request merged**